neuromation/api/config.py

```diff
@@ -449,3 +465,35 @@ def _load_file(filename: Path) -> Mapping[str, Any]:
     config = toml.load(filename)
     _validate_user_config(config, filename)
     return config
+
+
+def _load_schema(db: sqlite3.Connection) -> Dict[str, str]:
+    cur = db.cursor()
+    schema = {}
+    cur.execute("SELECT type, name, sql from sqlite_master")
+    for row in cur:
```

Comment:
You can also unpack a row:
```suggestion
    for type, name, sql in cur:
```